src/tester.rs

```diff
@@ -358,47 +362,47 @@ impl Tester {
         let mut return_wasm: Option<i32> = None;
         match self.function_call.remove(0) {
             FunctionCall::Start() => {
-                let _start = self
-                    .instance
-                    .get_func("_start")
-                    .ok_or(anyhow::format_err!(
-                        "Error: failed to find `_start` function export"
-                    ))?
-                    .get0::<()>()?;
-                println!("[host->vm] _start()");
-                _start()?;
+                println!("[host->vm] _initialize()");
+                self.instance
+                    .get_typed_func::<(), ()>(&mut self.store, "_initialize")
+                    .or(Err(anyhow::format_err!(
+                        "Error: failed to find `_initialize` function export"
+                    )))?
+                    .call(&mut self.store, ())?;
```

Comment:
I don't think this should be fixed that way, but unsure how to best support the `_start` along `_initialize` in _this particular_ context:

 - [ ] Silently fallback?
 - [ ] Add a param to the method to explicitly allow for falling back?
 - [ ] Add a `call_initialize`method?

Reply:
Looks like the first option might be the better one then...